packages/dullahan-runner-aws-lambda/src/DullahanRunnerAwsLambda.ts

```diff
@@ -8,8 +8,8 @@ import {
     DullahanClient,
     DullahanError,
     DullahanFunctionEndCall,
-    DullahanRunner,
-    DullahanTestEndCall,
+    DullahanRunner, DullahanTest,
+    DullahanTestEndCall, hasProperty,
```

Comment:
```suggestion
    DullahanRunner,
    DullahanTest,
    DullahanTestEndCall,
    hasProperty,
```

Looks like auto import did not do it's magic properly.

Reply:
Yeah I don't know why.... but it seems that a year ago I disabled ESLint (or at least... all rules) and I don't really remember why anymore.